package.json

```diff
@@ -35,7 +39,7 @@
     "mocha": "^10.0.0",
     "mocha-lcov-reporter": "^1.3.0",
     "multibase": "^3.1.0",
-    "multicodec": "^2.0.1",
+    "multicodec": "^3.2.1",
```

Comment:
We will need to move off of this package since it's deprecated and will not receive the new p*-priv multicodec codes. This change should be addressed in a separate PR and completed before release.

Reply:
Created Issue #4 to address this `FIXME` comment